File: analyze.py
import re, os, sys

def analyze(fname):
    if not os.path.exists(fname):
        return
    with open(fname, 'r', encoding='utf-8', errors='ignore') as f:
        text = f.read()
    
    print(f"=== {fname} ({len(text)} bytes) ===", flush=True)
    
    # Extract strings matching /something
    paths = set(re.findall(r'\"(/[a-zA-Z0-9_\-/\?=&]+)\"', text) + re.findall(r'\'(/[a-zA-Z0-9_\-/\?=&]+)\'', text))
    interesting = [p for p in paths if any(k in p.lower() for k in ['venue', 'slot', 'book', 'auth', 'sport', 'city', 'search', 'cart', 'pay', 'user', 'otp', 'token', 'checkout', 'v1', 'v2', 'api'])]
    print("  Paths:", interesting[:30], flush=True)

analyze('venue_search.js')
analyze('venue_detail.js')

File: analyze.js
const fs = require('fs');

function analyzeFile(filename) {
  if (!fs.existsSync(filename)) return;
  const content = fs.readFileSync(filename, 'utf8');
  const parts = content.split(/["']/);
  const matches = parts.filter(p => 
    p.length > 2 && p.length < 100 && 
    (p.includes('venue') || p.includes('slot') || p.includes('book') || p.includes('cart') || p.includes('api') || p.includes('sport') || p.includes('http'))
  );
  console.log(`=== ${filename} ===`);
  const unique = Array.from(new Set(matches));
  console.log('Matches:', unique.slice(0, 50));
}

['venue_detail.js', 'venue_search.js', 'app.js', 'commons.js', 'chunk80e.js'].forEach(analyzeFile);
